script.js

```diff
@@ -61,13 +113,15 @@ document.addEventListener("click", function(){
     var box8 = boxList[7].innerText;
     var box9 = boxList[8].innerText;
 
-    //  checkRow
-    if (check(box1, box2, box3) === "yes" || check(box4, box5, box6) === "yes" || check(box7, box8, box9) === "yes") {
-        if(!alert("Someone Won! \nClick ok to restart.")){window.location.reload();}
-    } else if (check(box1, box4, box7) === "yes" || check(box2, box5, box8) === "yes" || check(box3, box6, box9) === "yes") {  // checkcolumn
-        if(!alert("Someone Won! \nClick ok to restart.")){window.location.reload();}
-    } else if (check(box1, box5, box9) === "yes" || check(box3, box5, box7) === "yes") {  // check diagonal
-        if(!alert("Someone Won! \nClick ok to restart.")){window.location.reload();}
-    }
+    console.log(playerTurn);
+
+        if (check(box1, box2, box3) === "yes" || check(box4, box5, box6) === "yes" || check(box7, box8, box9) === "yes") {  // check row
+            if(!alert(`${winner()} won! \n`));
```

Comment:
Would recommend using an array to check for the winning combinations. And perhaps should have an array for each row of the tic tac toe game so that the game has scale. i.e. 4x4 up to n x n type tic tac toe.